admin-dev/themes/new-theme/template/controllers/orders/page_header_toolbar.tpl

```diff
@@ -22,11 +22,18 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *}
-<div class="header-toolbar">
-  <div class="container-fluid">
-      {literal}
-        {% block page_header_toolbar %}{% endblock %}
-      {/literal}
-  </div>
-  {hook h='displayDashboardTop'}
-</div>
+{extends file='page_header_toolbar.tpl'}
+
+{block name=pageTitle}
+  {if isset($use_regular_h1_structure) == false}
```

Comment:
Simple question

`{if !isset($use_regular_h1_structure)}` or `{if isset($use_regular_h1_structure) == false}`, one of these is best practice or we can use both ?

Reply:
Pierre told me